build_tools/cmake/build_host_tools.sh

```diff
@@ -13,27 +13,16 @@
 set -xeuo pipefail
 
 ROOT_DIR="${ROOT_DIR:-$(git rev-parse --show-toplevel)}"
-cd "${ROOT_DIR?}"
-
 BUILD_DIR="${1:-${IREE_HOST_BUILD_DIR:-build-host}}"
 INSTALL_DIR="${INSTALL_DIR:-${BUILD_DIR}/install}"
 CMAKE_BIN="${CMAKE_BIN:-$(which cmake)}"
 IREE_ENABLE_ASSERTIONS="${IREE_ENABLE_ASSERTIONS:-OFF}"
 
-"${CMAKE_BIN?}" --version
-ninja --version
-
-# --------------------------------------------------------------------------- #
-if [[ -d "${BUILD_DIR}" ]]; then
-  echo "${BUILD_DIR} directory already exists. Will use cached results there."
-else
-  echo "${BUILD_DIR} directory does not already exist. Creating a new one."
-  mkdir -p "${BUILD_DIR}"
-fi
+cd "${ROOT_DIR}"
```

Comment:
If we change the CWD, wondering if we need to first apply realpath on all input variables? Like the `INSTALL_DIR` here.

This will most likely only affect the manual run, since the worflows always run from the git root.

Reply:
Ah good point. Given that this is really intended for CI maybe rather than trying adding realpath to hopefully make everything work locally, we should instead just enforce that it's run from the git root?

Reply:
Which can be done pretty easily by just not attempting to find the git root and assuming that the helper scripts are at their relative paths :-P

Reply:
SGTM to enforce run from git root